components/shared/FormInput.js

```diff
@@ -48,7 +49,7 @@ const getSelectStyles = () => {
   };
 };
 
-export const FormTextInput = styled.input`
+export const FormTextInput = styled(Field)`
```

Comment:
@kburnell calling this out for you, I changed this to `Field` here so I could get the field level validate. I didn't seen any fall out from the change but want you to be aware.